priv/rag/ingest_text.py

```
@@ -0,0 +1,500 @@
+                    ),
+                    timestamp,
+                    timestamp,
+                ),
+            ).fetchone()[0]
+
+            chunk_index = 0
+            for chunk_batch in batches(chunks, args.batch_size):
+                contents = [chunk["content"] for chunk in chunk_batch]
+                embeddings = embedder.embed_documents(contents)
+
+                for chunk, embedding in zip(chunk_batch, embeddings):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)